### DISCUSSION TITLE
Screenshot are dimmed on my Mac Mini 2018 (the screen is a HP Z27)

### DISCUSSION THREAD

**Discussion by clementquentin** [MonitorControl/MonitorControl#1189] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hi,

When I dim my screen (HP Z27) on my Mac Mini 2018, if I make a screenshot (cmd+Shift+number), the resulting image is also dimmed.

This started yesterday and I didn't change anything to my config, or to the way I use Monitor Control.

The screen is connected through HDMI (if that matters).

I hope someone here will be able to help me !

Thanks,

CQ

PS : I just found out that if I try to make a screenshot of just a specific window (cmd+Shift+5, I tried to make one of what I'm typing here to join as an example), it only shows grey. A fullscreen screenshot (cmd+Shift+3) or partial screenshot (cmd+Shift+4) still shows what's on screen though. I attached two images to this post.

<img width="1920" alt="Capture d’écran 2022-09-22 à 21 14 08" src="https://user-images.githubusercontent.com/114185560/191832424-e99ee83c-b506-49e6-949d-045463cf7e57.png">
<img width="869" alt="Capture d’écran 2022-09-22 à 21 14 23" src="https://user-images.githubusercontent.com/114185560/191832433-1b81248a-ce39-4565-bf40-48592af8f487.png">

### Steps to reproduce

Make a screenshot with Monitor Control on.

### Expected behavior

Screenshots are dimmed at the same level as Monitor Control dims the screen.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: Mac Mini 2018
- MonitorControl version: 4.1.0
- Monitor(s): HP Z27
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment, marked as answer]
This happens when you are using overlay (shade) dimming instead of DDC (does not work via HDMI on the 2018 Mini) or gamma table manipulation.

<img width="842" alt="Screenshot 2022-09-22 at 21 38 39" src="https://user-images.githubusercontent.com/37590873/191836124-75ed8780-2111-4748-acbf-8512cb21999d.png">

**Comment by jmreeves11** [comment]
I have a follow up question. The same issue is happening for me. Dell monitor connected via HDMI-to-USBC to a MacBook pro with M1. Screenshots are dull/faded when MonitorControl is on. Would prefer to not have to quit monitor control every time I need to take a screenshot. Is there some kind of work-around? Are there certain display settings I should have toggled on/off in my screenshot attached?

Also, as a non tech person, what does DDC stand for and what does it mean (tried googling, didn't really understand), and what does gamma table manipulation mean?

Thanks so much.

<img width="721" alt="Screen Shot 2022-12-11 at 7 34 25 AM" src="https://user-images.githubusercontent.com/119472483/206903921-5d7bdfca-e757-476b-bf24-cc6979d496f8.png">

**Comment by dsmiddy19** [comment]
Bumping this up. Can i request a feature to provide an option to users to disable monitorcontrol when a screenshot is taken and then enable it right after the screenshot is taken? That would solve this issue. thoughts?

**Comment by dsmiddy19** [comment]
bumping again. any thoughts?
